lib/private/machines/create-each-record.js

```diff
@@ -97,6 +97,7 @@ module.exports = {
         return exits.success();
       }//-•
 
+      var insertedIds = Object.values(nativeResult.insertedIds);
 
```

Comment:
I think we should paste in (and slightly tweak) the assertion from here: https://github.com/balderdashy/sails-mongo/pull/499/files#diff-962fecaf7496f0dc0f4c49842e1e52f7035018902cd4530b86833883185c0166L106

that way we'll know if something changes in Mongo in the future and keep things just as easy to follow for migrating to future mongo driver releases

Reply:
You are right @mikermcneil. I just added the assertions and tweaked it to match what we are checking.